src/pages/charging-stations/index.tsx

```diff
@@ -32,29 +31,51 @@ import {
 } from './queries';
 import { TriggerMessageForEvseCustomAction } from '../../message/trigger-message';
 import { ChargingStationsListQuery } from '../../graphql/types';
-import { GenericViewState } from '../../model/enums';
+import { useDebounce } from '../../hooks/useDebounce';
+import { GenericViewState, SelectionType } from '@enums';
 
 const { Panel } = Collapse;
 const { Sider, Content } = Layout;
 
-export const ChargingStationsView: React.FC = () => {
+const styles = {
```

Comment:
why dont we just put these styles into scss and set a class name?

Reply:
It is on my todo list.